src/luarocks/api/doc.lua

```diff
@@ -109,4 +109,27 @@ function doc.doc(name, version, tree)
    return docdir, nil, files 
 end
 
+function doc.open_doc(docdir, docfile)
+   if not docdir then
+      return nil, "No doc directory given as argument."
+   end
+   
+   if not docfile then
+      return nil, "No doc files given as argument."
+   end
+
+   local pathname = dir.path(docdir, docfile)
+   if not fs.browser then
+      fs.init()
+   end
+   local ok = fs.browser(pathname)
+   if not ok and not docfile:match("%.html?$") then
+      local fd = io.open(pathaname, "r")
```

Comment:
there's a typo here (pathaname) — (which means this code path is untested!)

Reply:
I forgot to push that in a hurry.